pkg/proxy/command.go

```diff
@@ -263,6 +283,119 @@ func ExtractCommandAction(ctx context.Context, command string, params RequestPar
 		return func(v *vehicle.Vehicle) error { return v.Unlock(ctx) }, nil
 	case "erase_user_data":
 		return func(v *vehicle.Vehicle) error { return v.EraseGuestData(ctx) }, nil
+	case "add_charge_schedule":
+		lat, err := params.getNumber("lat", true)
```

Comment:
nit: any harm in using the full word for the interface? I suppose consistency?: https://developer.tesla.com/docs/fleet-api/endpoints/vehicle-commands#navigation-gps-request

Reply:
Three endpoints already use `lon`, so going to keep with the precedence.